lib/api/reference.js

```diff
@@ -168,7 +168,7 @@ module.exports = {
 
     var promise = client.request({
       url: url,
-      method: 'GET',
+      method: 'POST',
```

Comment:
@alexbor the POST reference/bank_details endpoint was renamed to reference/bank_details/find, so you will need to update that. See 

https://github.com/CurrencyCloud/currencycloud-swagger/pull/247/files#diff-503fd62126a95472639899ef0c06546ed50daa650c1e5cde73cd3ba04f8e85acR12852